intern/remote/boss/remote.py

```diff
@@ -866,13 +867,24 @@ def get_cutout(self, resource, resolution, x_range, y_range, z_range, time_range
                 z_range (list[int]): z range such as [10, 20] which means z>=10 and z<20.
                 time_range (optional [list[int]]): time range such as [30, 40] which means t>=30 and t<40.
                 id_list (optional [list[int]]): list of object ids to filter the cutout by.
-                no_cache (optional [boolean]): specifies the use of cache to be True or False. 
+                access_mode (optional [Enum]): Identifies one of three cache access options:
+                    cache = Will check both cache and for dirty keys
+                    no_cache = Will skip cache check but check for dirty keys
+                    raw = Will skip both the cache and dirty keys check
+
+                TODO: Add mode to documentation
 
             Returns:
                 (numpy.array): A 3D or 4D (time) numpy matrix in (time)ZYX order.
 
             Raises:
                 requests.HTTPError on error.
             """
-
-            return self._volume.get_cutout(resource, resolution, x_range, y_range, z_range, time_range, id_list, no_cache, **kwargs)
+            if no_cache != None:
```

Comment:
Not sure if this is the correct way to check for None. I normally use: `if no_cache is not None:`

Reply:
According to https://stackoverflow.com/questions/3965104/not-none-test-in-python It seems both in this case will be correct. They do point out a benefit to using your way. Which is enough to convince me to switch to yours.